region/norfair/west/Crumble Shaft.json

```diff
@@ -470,13 +470,24 @@
       "requires": [
         "h_heatProof",
         {"or": [
-          "canCrumbleJump",
-          "h_canUseSpringBall"
+          {"and": [
+            "canCrumbleJump",
+            "canConsecutiveWalljump"
+          ]},
+          {"and": [
+            "h_canUseSpringBall",
+            "canDodgeWhileShooting"
```

Comment:
dodgewhileshooting could go on both options, to avoid getting hit on a descent. This room might be a bit too annoying to farm in Basic/Medium anyway